docs/book/src/topics/container-networking-interface.md

```diff
@@ -0,0 +1,21 @@
+# Container Networking Interface
+
+This document aims to provide steps to configure your cluster using the below CNI solutions
+
+- [Azure CNI v1](#azure-container-networking-interface-v1)
+
+## Azure Container Networking Interface v1
+
+As of writing this document, Azure CNI needs to be installed in the following steps below. 
+
+- Bring up your workload cluster with desired number of Public IP Configs
+- Update Azure VNet Config in `/etc/cloud/cloud.cfg.d/` with the following contents placed in `15-azure-vnet.cfg` file
```

Comment:
What reads this configuration file? Perhaps it could be shipped via ConfigMap together with `azure-cni` daemonset?

Reply:
resolving this since https://github.com/kubernetes-sigs/image-builder/pull/1090 has merged.